bin/security/setupSecurityAccount.sh

```diff
@@ -3,19 +3,17 @@
 option="${1}"
 
 BB_ROOT_DIR=$(cd $(dirname "$0"); cd ..; pwd)
-COMPOSE=$BB_ROOT_DIR/$(ls $BB_ROOT_DIR | awk '/docker-compose/ && !/test-tools/')
+MAKEFILE=$BB_ROOT_DIR/Makefile
 
 case ${option} in
   -useradd)
   echo "Info: Add security account."
-  OT=$(docker-compose -f ${COMPOSE} run --rm \
-    --entrypoint /edgex/security-proxy-setup edgex-proxy --init=false --useradd=geneva --group=admin | grep '^the access token for')
-  export TOKEN=$(echo ${OT} | sed 's/.*: \([^.]*\.[^.]*\.[^.]*\).*/\1/')
+  export TOKEN=`make -f ${MAKEFILE} get-token ${USE_ARM64} geneva`
```

Comment:
Need `[ "$(uname -m)" != "x86_64" ] && USE_ARM64="arm64"` for ${USE_ARM64}

Reply:
Fixed